src/style/app.module.css

```diff
@@ -2996,10 +3053,22 @@ form > input {
   left: 1rem;
 }
 
+/* Define your custom button styles */
 .langChangeBtnStyle {
+  --bs-btn-active-bg: var(--light-blue);
+  --bs-btn-active-border-color: var(--light-blue);
+  --bs-btn-active-color: var(--dark-blue);
   width: 7.5rem;
   height: 2.2rem;
   padding: 0;
+  color: var(--dark-blue);
+  border-color: var(--light-blue);
+  background-color: transparent; /* Default transparent background */
+}
+
+.langChangeBtnStyle:hover {
+  background-color: var(--light-blue) !important;
+  border-color: var(--light-blue) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`